nixos/modules/security/acme.nix

```diff
@@ -52,6 +52,36 @@ let
         '';
       };
 
+      systemd = mkOption {
+        description = "submodule example";
+        default = {};
+        type = with types; submodule {
+          options = {
+
+            reload = mkOption {
+              type = types.listOf types.str;
+              default = [];
+              example = [ "postfix.service" ];
```

Comment:
`postfix.service` is used in both examples. Which one is needed for postfix and can we find a different common example for the other-one?

Reply:
my main goal is `postfix.service` but maybe there are other use-cases as well.

Reply:
we could use `dovecot.service` for instance